src/components/attribute-form/AttributeForm.tsx

```diff
@@ -75,7 +81,7 @@ export const AttributesForm = ({
   }, [fields]);
 
   return (
-    <FormAccess role="manage-realm" onSubmit={handleSubmit(save)}>
+    <FormAccess role="manage-realm" onSubmit={handleSubmit(save!)}>
```

Comment:
Using a non-null assertion here might cause errors at runtime. Instead you can check the value before assigning it:
```suggestion
    <FormAccess role="manage-realm" onSubmit={save ? handleSubmit(save) : undefined}>
```